src/module/dice.js

```diff
@@ -262,9 +262,9 @@ export class DiceSFRPG {
             const itemContext = rollContext.allContexts['item'];
             const htmlData = [{ name: "rollNotes", value: itemContext?.system?.rollNotes }];
 
-            let useCustomCard = game.settings.get("sfrpg", "useCustomChatCards");
+            let useCustomCard = game.settings.get("sfrpg", "useCustomChatCards") && chatMessage;
```

Comment:
This check is needless given lines 267, 288, 757 and 783 check for `chatMessage` anyway

Reply:
done. thanks for the hint.